### ISSUE TITLE
Python 3 💀

### ISSUE THREAD

**Issue by hynek** [glyph/python-docstring-mode#11]
When my Emacs has activated an Python 3 virtualenv, python-docstring-mode stops working:

``` python
"""(most recent call last):
  File "/Users/hynek/.emacs.d/docstring_wrap.py", line 543, in <module>
    indata = sys.stdin.read().decode("utf-8")
AttributeError: 'str' object has no attribute 'decode'
"""
```

(there’s a few more errors but it seems manageable)

**Comment by glyph** [comment]
Huh. I tried to be mindful of str/bytes stuff, sorry :-.  What even is a stdin.

**Comment by glyph** [comment]
The virtualenv issue should be addressed by #14 and the broader philosophical issue is at least touched on by #12 .

**Comment by glyph** [comment]
Going to mark as fixed by #14 for now.
